src/rand_distr.rs

```diff
@@ -0,0 +1,126 @@
+use rand::Rng;
+use rand_distr::{uniform::UniformFloat, Distribution};
+
+macro_rules! impl_distribution_via_f32 {
+    ($Ty:ty, $Distr:ty) => {
+        impl Distribution<$Ty> for $Distr {
+            fn sample<R: Rng + ?Sized>(&self, rng: &mut R) -> $Ty {
```

Comment:
The `rand::Rng` trait is used in this impl Distribution here. Unfortunately rand_distr doesn't re-export Rng, so thats the only use of it.

This would mean users would always have to use `--features rand,rand_distr` together. Thoughts?